requirements/pip.txt

```diff
@@ -92,7 +92,7 @@ textblob==0.15.3
 
 django-annoying==0.10.5
 # Messages-extends needs a fix for Django 2.2 which isn't released (0.6.1 isn't out yet)
-git+https://github.com/AliLozano/django-messages-extends.git@b1e4f4c#egg=django-messages-extends==0.6.1-dev
+git+https://github.com/AliLozano/django-messages-extends.git@5e1cd27#egg=django-messages-extends==0.6.2-dev
```

Comment:
0.6.1 still looks to be the next release. If you are using `0.6.2` as a way to increment, 0.6.2-dev1 or something similar would be less confusing later.

Reply:
I've already published it as 0.6.1 into pip.